### ISSUE TITLE
Module not found: Error Can't resolve SOME_MODULE ../css-loader/dist/runtime

### ISSUE THREAD

**Issue by nmccready** [webpack-contrib/css-loader#1057]
Howdy, this is entirely a last result at already spending several hours searching SO, GitHub, and debugging code to literally questioning my existence.

Anyway download this repo as a minimal example: https://github.com/nmccready/webpack-ts-react

- `yarn install`

then .. run the headache below.
```
~/code/nmccready/webpack-ts-react
❯ yarn build --display-error-details --verbose --debug
yarn run v1.21.1
$ node -r esm ./node_modules/.bin/webpack --display-error-details --verbose --debug
Hash: db71985b609a7625b9b8
Version: webpack 4.42.0
Time: 10471ms
Built at: 03/20/2020 11:51:01 AM
                Asset       Size  Chunks                   Chunk Names
 CssModule/index.d.ts   83 bytes          [emitted]
HelloWorld/index.d.ts  106 bytes          [emitted]
           index.d.ts   59 bytes          [emitted]
             index.js   18.5 KiB    main  [emitted]        main
         index.js.map   17.5 KiB    main  [emitted] [dev]  main
Entrypoint main = index.js index.js.map
chunk {main} index.js, index.js.map (main) 8.79 KiB [entry] [rendered]
    > ./src/index.ts main
 [./node_modules/@teamsupercell/typings-for-css-modules-loader/src/index.js?!./node_modules/css-loader/dist/cjs.js?!./src/CssModule/index.module.css] ./node_modules/@teamsupercell/typings-for-css-modules-loader/src??ref--5-1!./node_modules/css-loader/dist/cjs.js??ref--5-2!./src/CssModule/index.module.css 394 bytes {main} [depth 3] [built]
     cjs require !!../../node_modules/@teamsupercell/typings-for-css-modules-loader/src/index.js??ref--5-1!../../node_modules/css-loader/dist/cjs.js??ref--5-2!./index.module.css [./src/CssModule/index.module.css] 2:26-197
 [./node_modules/postcss-loader/src/index.js!./src/HelloWorld/index.css] ./node_modules/postcss-loader/src!./src/HelloWorld/index.css 74 bytes {main} [depth 3] [built]
     cjs require !!../../node_modules/postcss-loader/src/index.js!./index.css [./src/HelloWorld/index.css] 2:26-97
 [./src/CssModule/index.module.css] 672 bytes {main} [depth 2] [built]
     harmony side effect evaluation ./index.module.css [./src/CssModule/index.tsx] 2:0-47
     harmony import specifier ./index.module.css [./src/CssModule/index.tsx] 3:71-80
 [./src/CssModule/index.tsx] 175 bytes {main} [depth 1] [built]
     [exports: CssModule]
     harmony side effect evaluation ./CssModule [./src/index.ts] 2:0-28
     harmony export imported specifier ./CssModule [./src/index.ts] 2:0-28
 [./src/HelloWorld/index.css] 572 bytes {main} [depth 2] [built]
     harmony side effect evaluation ./index.css [./src/HelloWorld/index.tsx] 2:0-21
 [./src/HelloWorld/index.tsx] 154 bytes {main} [depth 1] [built]
     [exports: HelloWorld]
     harmony side effect evaluation ./HelloWorld [./src/index.ts] 1:0-29
     harmony export imported specifier ./HelloWorld [./src/index.ts] 1:0-29
 [./src/index.ts] 59 bytes {main} [depth 0] [built]
     [exports: HelloWorld, CssModule]
     single entry ./src/index.ts  main
 [react] external "react" 42 bytes {main} [depth 2] [built]
     harmony side effect evaluation react [./src/CssModule/index.tsx] 1:0-26
     harmony import specifier react [./src/CssModule/index.tsx] 3:31-36
     harmony side effect evaluation react [./src/HelloWorld/index.tsx] 1:0-26
     harmony import specifier react [./src/HelloWorld/index.tsx] 3:32-37
     + 2 hidden modules

LOG from webpack.buildChunkGraph.visitModules
<t> prepare: 0.768182ms
<t> visiting: 0.1585ms

ERROR in ./node_modules/css-loader/dist/runtime/api.js
Module not found: Error: Can't resolve './CssModule' in '/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime'
resolve './CssModule' in '/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime'
  using description file: /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/package.json (relative path: ./dist/runtime)
    using description file: /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/package.json (relative path: ./dist/runtime/CssModule)
      no extension
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule doesn't exist
      .js
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.js doesn't exist
      .jsx
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.jsx doesn't exist
      .ts
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.ts doesn't exist
      .tsx
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.tsx doesn't exist
      .json
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.json doesn't exist
      .css
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.css doesn't exist
      as directory
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule doesn't exist
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.js]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.jsx]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.ts]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.tsx]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.json]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/CssModule.css]
 @ ./node_modules/css-loader/dist/runtime/api.js 2:0-28 2:0-28
 @ ./node_modules/@teamsupercell/typings-for-css-modules-loader/src??ref--5-1!./node_modules/css-loader/dist/cjs.js??ref--5-2!./src/CssModule/index.module.css
 @ ./src/CssModule/index.module.css
 @ ./src/CssModule/index.tsx
 @ ./src/index.ts

ERROR in ./node_modules/css-loader/dist/runtime/api.js
Module not found: Error: Can't resolve './HelloWorld' in '/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime'
resolve './HelloWorld' in '/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime'
  using description file: /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/package.json (relative path: ./dist/runtime)
    using description file: /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/package.json (relative path: ./dist/runtime/HelloWorld)
      no extension
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld doesn't exist
      .js
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.js doesn't exist
      .jsx
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.jsx doesn't exist
      .ts
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.ts doesn't exist
      .tsx
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.tsx doesn't exist
      .json
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.json doesn't exist
      .css
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.css doesn't exist
      as directory
        /Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld doesn't exist
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.js]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.jsx]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.ts]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.tsx]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.json]
[/Users/nem/code/nmccready/webpack-ts-react/node_modules/css-loader/dist/runtime/HelloWorld.css]
 @ ./node_modules/css-loader/dist/runtime/api.js 1:0-29 1:0-29
 @ ./node_modules/@teamsupercell/typings-for-css-modules-loader/src??ref--5-1!./node_modules/css-loader/dist/cjs.js??ref--5-2!./src/CssModule/index.module.css
 @ ./src/CssModule/index.module.css
 @ ./src/CssModule/index.tsx
 @ ./src/index.ts
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Notice the following comments:
- https://github.com/nmccready/webpack-ts-react/blob/master/webpack.config.js#L30-L31
- https://github.com/nmccready/webpack-ts-react/blob/master/webpack.config.js#L47

Bottom line is anytime I actually need css-loader webpack blows up. My guess this is a dumb error on my part, but I am at a total loss.

**Comment by nmccready** [comment]
Also note that I have tried reverting to 1.X, 2.X versions of css-loader all having similar errors.

**Comment by evilebottnawi** [comment]
Yes, the problem from `typings-for-css-modules-loader`, if advises above doesn't help feel free to open a new issue in `typings-for-css-modules-loader` repo, thanks for issue and feel free to leave feedback
